libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -893,7 +893,6 @@ - (void)authAtLaunch
     
     SFUserAccountManagerSuccessCallbackBlock successBlock = ^(SFOAuthInfo *authInfo,SFUserAccount *userAccount) {
         [SFSDKCoreLogger i:[self class] format:@"Authentication (%@) succeeded.  Launch completed.", authInfo.authTypeDescription];
-        [SFUserAccountManager sharedInstance].currentUser = userAccount;
```

Comment:
What was that line for?

Reply:
SFUserAccountManager:loginwithCompletion should set it. No need to explicitly set it anymore.